src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -99,8 +99,8 @@
  *   -I input.bam \
  *   -O output.g.vcf.gz \
  *   -ERC GVCF \
- *   -G Standard \
- *   -G AS_Standard
+ *   -G StandardAnnotation \
```

Comment:
Looks like some extra code was included in this PR.  Can you revert this change?

Reply:
Okay Fixed!